### PR TITLE
Remove relaymq dependency and RelayMQDriver and its config

### PR DESCRIPTION
**Pull request by SenRamakri**
This is the change to remove RelayMQ driver and dependency from Maestro.

### CODE REVIEW

**Comment by SenRamakri** [comment]
I have not done system testing with this change yet. I'm starting on that now, but the code review can start now.

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -138,14 +138,6 @@
 			"ImportPath": "github.com/dsnet/compress/internal/prefix",
 			"Rev": "cc9eb1d7ad760af14e8f918698f745e80377af4f"
 		},
-		{
-			"ImportPath": "github.com/edhemphill/relaymq",
```
Comment (cmonr) [inline Godeps/Godeps.json]: 🎉

File: vendor/github.com/armPelionEdge/greasego/deps/include/version.h [hunk 2]
```diff
@@ -26,4 +26,4 @@
 */
 
 #define GREASE_LIB_VERSION "0.1.2"
-#define GREASE_LIB_COMMIT "5b578dd"
+#define GREASE_LIB_COMMIT "6cf0b3e"
```
Comment (cmonr) [inline vendor/github.com/armPelionEdge/greasego/deps/include/version.h]: Should this file be reverted?

Comment (SenRamakri) [reply vendor/github.com/armPelionEdge/greasego/deps/include/version.h]: Yes, I'll revert this. Thanks for pointing this out.

**Comment by edhemphill** [comment]
@SenRamakri - as far as I can tell checktime.go should be fine. Been a while since I wrote that code. Let's check to make sure its working to integration cloud.

**Comment by SenRamakri** [comment]
@edhemphill @cmonr - I have tested these changes and confirmed that gateway is successfully connecting and registering with the pelion cloud(os2). I think we can merge the changes now.

**Pull request merged**